modules/appnexusBidAdapter.js

```diff
@@ -59,6 +60,23 @@ export const spec = {
         .forEach(param => userObj[param] = userObjBid.params.user[param]);
     }
 
+    const appDeviceObjBid = find(bidRequests, hasAppDeviceInfo);
```

Comment:
@aneuway2 Just to confirm a use-case - do you foresee the chance that multiple adUnits for `appnexus` would ever have differing values for this `device`/`geo` fields?  

This `find` grabs the first matching result and uses that for the value.  If you had multiple adUnits with differing sets of values, only the first set of those values would be used - potentially skewing the request that gets generated.

I'm not sure this would happen, but I wanted to clarify just in case.